packages/vue/tsup.config.ts

```diff
@@ -13,8 +14,10 @@ export default defineConfig(() => {
     bundle: true,
     sourcemap: true,
     minify: false,
-    dts: true,
+    dts: false,
     esbuildPlugins: [
+      // Adds .vue files support
+      vuePlugin() as EsbuildPlugin,
```

Comment:
This `vuePlugin` transforms `.vue` files to JavaScript. We also turned off `dts` generation of tsup so that the `vue-tsc` module will generate it instead